packages/react/src/components/Toolbar/ToolbarMenuItem.tsx

```diff
@@ -35,12 +35,21 @@ export interface ToolbarMenuItemProps
   /** A toolbar item can be active. */
   active?: boolean
 
+  /** Item can show check indicator if selected. */
+  checked?: boolean
```

Comment:
sorry that I missed it before - for toggle in the toolbar we are using toggleButtonBehavior, there the prop is called `active`. That way it would even be consistent with the toolbar menu radio.